Backend/src/views/mails/invitation_to_institution/text.txt

```diff
@@ -0,0 +1,8 @@
+Invitation to a team
+
+Hey,
+We want to inform you that you have invited {{firstname}} to the institution {{institution}}.
```

Comment:
Same issue as with Isaac in #31  (see my comments) 
It doesnt make too much sense if the HTML file has different content than the txt file, as both are sent to the same person. 
I would approve this anyways (also see last comment #31)